nxdrive/gui/api.py

```diff
@@ -443,6 +444,23 @@ def default_server_url_value(self) -> str:
         """Make daily job better for our developers :)"""
         return getenv("NXDRIVE_TEST_NUXEO_URL", "")
 
+    @pyqtSlot(str, result=bool)
+    def free_disk_space_under_limit(self, path: str) -> bool:
+        remaining_space = shutil.disk_usage(Path(path).parent).free
+
+        # 10737418240 = 10GiB
+        return remaining_space < 10737418240
+
+    @pyqtSlot(str, result=str)
+    def get_used_disk_space(self, path: str) -> str:
+        used_space = shutil.disk_usage(path).used
+        return sizeof_fmt(used_space)
+
+    @pyqtSlot(str, result=str)
+    def get_free_disk_space(self, path: str) -> str:
+        remaining_space = shutil.disk_usage(Path(path).parent).free
```

Comment:
Do we need to use the parent? I think it is good to use the `path` directly, no?